pkgs/by-name/go/goshs/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildGoModule rec {
+  pname = "goshs";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "patrickhener";
+    repo = "goshs";
+    tag = "v${version}";
+    hash = "sha256-J74nUH3iBYDytXCswIexvShG5+zJEDX19saJ3ftLYhI=";
+  };
+
+  vendorHash = "sha256-6UaKh2UUyGqOriAaMhiEFr20R4W/ZWxQXtXnm/BVHOc=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  meta = {
+    description = "Simple, yet feature-rich web server written in Go";
+    homepage = "https://goshs.de";
+    license = lib.licenses.mit;
+    platforms = lib.platforms.all;
+    # --- FAIL: TestGetIPv4Addr (0.00s)
+    #     utils_test.go:62: route ip+net: no such network interface
+    badPlatforms = lib.platforms.darwin;
```

Comment:
Hnnrgh, feels weird to mark all Darwin builds as broken when only one test is failing. Should we just disable that test on Darwin? (Also, just a hunch, but given this is about networking and only happens on Darwin, try setting `__darwinAllowLocalNetworking` to true)